### PR TITLE
Log favicon state update while viewing logs

### LINKED ISSUE #639: [UI] update status favicon while viewing logs
When I'm viewing a task, the icon in the tab shows the status of the build (running, passed, failed) even if I change to a different tab.
It would be awesome if the status was still shown in the icon, even if I was viewing a log file [like this one](https://taskcluster-web.netlify.com/tasks/JU3ALKFIQTuN7kaTuTJ5KQ/runs/0/logs/live/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FJU3ALKFIQTuN7kaTuTJ5KQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log)

### PR DESCRIPTION
**Pull request by Shubhamchinda**
Fixes #639
@helfi92 

**After fix :**

Screenshot (195)

### CODE REVIEW

**Comment by Shubhamchinda** [comment]
Thanks a lot!
 
> I think one solution would be to have the `views/TaskLog` query for the the task run. In order to do this, we would need to create a `task.graphql` file in `views/Tasks/TaskLog/` and have a graphql request that fetches the task state. To use graphql in a view, you would need to use the `@graphql` decorator.

I was also trying to get the state of TaskLog in the ``` views/Task/TaskLog``` but couldn't figure out a way, then came up with this. It was working fine in my system even if I view the tasklog directly probably because of the cache.

Anyway, I'll update the changes asap.
Could you please provide some demo TaskId for testing?

Once again Thanks for explaining. :)

**Comment by helfi92** [comment]
> Could you please provide some demo TaskId for testing?

Hy2mxPUVSR214uNTq6rPCA
E61qVAV7RSuc09yGNYwyqw

Got them from https://treeherder.mozilla.org/#/jobs?repo=autoland by clicking on one of those symbols then clicking on the task ID

Screen Shot 2019-07-22 at 2 14 32 PM

**Comment by Shubhamchinda** [comment]
Hey @helfi92 , I've updated the patch with graphql and now the favicon is updating.

Although, there is one issue, the updated favicon is is showing after 1 sec, maybe because of the graphql request. Could you please help me here on how can I show the updated favicon state instantly after the page loads.

Thanks! :)

Ps : Please let me know if the patch requires any other changes too.

File: ui/src/views/Tasks/TaskLog/index.jsx [hunk 1]
```diff
@@ -21,13 +24,24 @@ import Link from '../../../utils/Link';
...
+})
 export default class TaskLog extends Component {
   render() {
     const { classes, match, stream } = this.props;
+    const { task } = this.props.data;
     const url = decodeURIComponent(match.params.logUrl);
 
     return (
       <Dashboard title="Log" disablePadding>
+        <Helmet state={task && task.status.state} />
```
Comment (helfi92) [inline ui/src/views/Tasks/TaskLog/index.jsx]: We should grab the state from the task run. `task.graphql` could be as simple as:

```graphql
query Task($taskId: ID!) {
  task(taskId: $taskId) {
    status {
      runs {
        state
      }
    }
  }
}

```

The run number can be grabbed using `this.props.match.params.runId`.

Comment (Shubhamchinda) [reply ui/src/views/Tasks/TaskLog/index.jsx]: @helfi92 Made the changes. :)

**Comment by Shubhamchinda** [comment]
> I think that's okay :)

Sometimes even more than 1 second.

**Comment by helfi92** [comment]
@allcontributors[bot] please add @Shubhamchinda for code!

**Comment by Shubhamchinda** [comment]
> This is really awesome! :)

Thanks a lot @helfi92 . I'll be around for more contibutions. Just tag me if you got anything for me. :)

**Pull request merged**